packages/java/tests/pom.xml

```diff
@@ -1,288 +1,311 @@
-        </plugin>
-        <!-- jetty plugin for those child modules that need it -->
-        <plugin>
-          <groupId>org.eclipse.jetty</groupId>
-          <artifactId>jetty-maven-plugin</artifactId>
-          <version>${jetty.version}</version>
-          <executions>
-            <execution>
-              <id>start-jetty</id>
-              <phase>pre-integration-test</phase>
-              <goals>
-                <goal>start</goal>
-              </goals>
-            </execution>
-            <execution>
-              <id>stop-jetty</id>
-              <phase>post-integration-test</phase>
-              <goals>
-                <goal>stop</goal>
-              </goals>
-            </execution>
-          </executions>
-          <configuration>
-            <webApp>
-              <containerIncludeJarPattern>^$
-              </containerIncludeJarPattern>
-            </webApp>
-            <httpConnector>
-              <port>${server.port}</port>
-            </httpConnector>
-            <stopPort>${server.stop.port}</stopPort>
-            <stopKey>foo</stopKey>
-            <stopWait>5</stopWait>
-          </configuration>
-        </plugin>
-        <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>properties-maven-plugin</artifactId>
-          <version>${properties-maven-plugin.version}</version>
-          <executions>
-            <execution>
-              <goals>
-                <goal>set-system-properties</goal>
-              </goals>
-              <configuration>
-                <properties>
-                  <vaadin.reuseDevServer>${vaadin.reuseDevServer}</vaadin.reuseDevServer>
-                  <vaadin.devmode.liveReload.enabled>${vaadin.devmode.liveReload.enabled}</vaadin.devmode.liveReload.enabled>
-                  <vaadin.allow.appshell.annotations>${vaadin.allow.appshell.annotations}</vaadin.allow.appshell.annotations>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>dev.hilla</groupId>
+                    <artifactId>hilla-maven-plugin</artifactId>
+                    <version>${project.version}</version>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>prepare-frontend</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-clean-plugin</artifactId>
+                    <version>3.2.0</version>
+                    <configuration>
+                        <filesets>
+                            <fileset>
+                                <directory>${project.basedir}</directory>
+                                <includes>
+                                    <include>node_modules/**</include>
+                                    <include>types.d.ts</include>
```

Comment:
Is the default `types.d.ts` file generated by flow working for React apps nowadays?

Reply:
Probably not but this part is not changed in this PR